flink-libraries/flink-table/src/test/java/org/apache/flink/table/api/java/stream/utils/Pojos.java

```diff
@@ -0,0 +1,61 @@
+package org.apache.flink.table.api.java.stream.utils;
```

Comment:
The Apache License header is missing. Please build PRs locally (or on Travis) to ensure that the build passes.